src/test/java/io/nats/client/impl/JetStreamManagementTests.java

```diff
@@ -236,9 +239,55 @@ public void testGetStreamInfo() throws Exception {
         runInJsServer(nc -> {
             JetStreamManagement jsm = nc.jetStreamManagement();
             assertThrows(JetStreamApiException.class, () -> jsm.getStreamInfo(STREAM));
-            createDefaultTestStream(jsm);
+
+            String[] subjects = new String[6];
+            for (int x = 0; x < 5; x++) {
+                subjects[x] = subject(x);
+            }
+            subjects[5] = "foo.>";
+            createMemoryStream(jsm, STREAM, subjects);
+
+            List<PublishAck> packs = new ArrayList<>();
+            JetStream js = nc.jetStream();
+            for (int x = 0; x < 5; x++) {
+                jsPublish(js, subject(x), x + 1);
+                PublishAck pa = jsPublish(js, subject(x), data(x + 2));
+                packs.add(pa);
+                jsm.deleteMessage(STREAM, pa.getSeqno());
+            }
+            jsPublish(js, "foo.bar", 6);
+
             StreamInfo si = jsm.getStreamInfo(STREAM);
             assertEquals(STREAM, si.getConfiguration().getName());
+            assertEquals(6, si.getStreamState().getSubjectCount());
+            assertNull(si.getStreamState().getSubjects());
```

Comment:
Ok, that seem to indicate that getStreamInfo() without the option is getting no list at all.

Reply:
I could have given an empty list though, but null is more consistent with the rest of the api.